File: wlxx-camunda/wlxx-camunda-provider/src/main/java/com/xzg/wlxx/camunda/config/SecurityConfiguration.java
package com.xzg.wlxx.camunda.config;

import org.springframework.context.annotation.Bean;
import org.springframework.context.annotation.Configuration;
import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
import org.springframework.security.config.annotation.web.configuration.WebSecurityCustomizer;
import org.springframework.security.web.util.matcher.AntPathRequestMatcher;

@Configuration
@EnableWebSecurity
public class SecurityConfiguration {

    @Bean
    public WebSecurityCustomizer webSecurityCustomizer() {
        return (web) -> {
            AntPathRequestMatcher[] antPathRequestMatcher = {
                    AntPathRequestMatcher.antMatcher("/**")
            };
            web.ignoring().requestMatchers(antPathRequestMatcher);
        };
    }
}

File: wlxx-camunda/wlxx-camunda-provider/src/main/kotlin/com/xzg/wlxx/camunda/config/SecurityConfiguration.kt
package com.xzg.wlxx.camunda.config

import org.springframework.context.annotation.Bean
import org.springframework.context.annotation.Configuration
import org.springframework.security.config.annotation.web.builders.WebSecurity
import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity
import org.springframework.security.config.annotation.web.configuration.WebSecurityCustomizer
import org.springframework.security.web.util.matcher.AntPathRequestMatcher

@Configuration
@EnableWebSecurity
class SecurityConfiguration {
    @Bean
    fun webSecurityCustomizer(): WebSecurityCustomizer {
        return WebSecurityCustomizer { web: WebSecurity ->
            val antPathRequestMatcher = arrayOf(
                AntPathRequestMatcher.antMatcher("/**")
            )
            web.ignoring().requestMatchers(*antPathRequestMatcher)
        }
    }
}
